### DISCUSSION TITLE
Can not login web console, get error, 401 401 Unauthorized

### DISCUSSION THREAD

**Discussion by qikang** [minio/minio#19330] [Q&A]
My environment info.

root@znlk-minio-01:~# mc admin user info local LoO02AsA7NIxkhUv --json
{
 "status": "success",
 "accessKey": "LoO02AsA7NIxkhUv",
 "policyName": "consoleAdmin,diagnostics,readwrite",
 "userStatus": "enabled"
}
root@znlk-minio-01:~# mc admin policy info local consoleAdmin --json
{
 "status": "success",
 "policy": "consoleAdmin",
 "policyInfo": {
  "PolicyName": "consoleAdmin",
  "Policy": {
   "Version": "2012-10-17",
   "Statement": [
    {
     "Effect": "Allow",
     "Action": [
      "admin:*"
     ]
    },
    {
     "Effect": "Allow",
     "Action": [
      "s3:*"
     ],
     "Resource": [
      "arn:aws:s3:::*"
     ]
    }
   ]
  },
  "CreateDate": "2024-03-22T09:19:12.373Z",
  "UpdateDate": "2024-03-22T09:19:12.373Z"
 },
 "isGroup": false
}
root@znlk-minio-01:~# minio --version 
minio version RELEASE.2024-03-15T01-07-19Z (commit-id=93fb7d62d80161480bac611c09a3fcab6f1196c0)
Runtime: go1.21.8 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2024 MinIO, Inc.
root@znlk-minio-01:~# 
root@znlk-minio-01:~# 

![image](https://github.com/minio/minio/assets/20736125/4486e5f8-5a40-47ae-8932-86bffedf4574)

**Comment by qikang** [comment]
offline deb file
wget https://dl.min.io/server/minio/release/linux-amd64/archive/minio_20240315010719.0.0_amd64.deb -O minio.deb
sudo dpkg -i minio.deb

**Reply by tty2** [reply]
I could make it work only when removed " symbol from password in vars

changed `MINIO_ROOT_PASSWORD="my_awesome_password"` to `MINIO_ROOT_PASSWORD=my_awesome_password`

**Comment by serdar-akyol** [comment]
Here I am also having the same problem. However, my MINIO_ROOT_USER and MINIO_ROOT_PASSWORD variables don't have any symbol or special character. Why do I have the same error?
